### PR TITLE
Add and remove task

### PR DESCRIPTION
**Pull request by audukehinde**
- Add Todo
- Edit todo
- Delete todo
- Save todo to localstorage

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,37 +1,23 @@
...
-      <div class="task"><p>${task.description}</p></div>
-    </div>
-    <i class="fa-solid fa-ellipsis-vertical"></i>
-  </div>`;
-    taskContent.insertAdjacentHTML('beforeend', html);
-  });
-};
+    todos.push(todo);
+    localStorage.setItem('todos', JSON.stringify(todos));
+    e.target.reset();
```
Comment (Reem-lab) [inline src/index.js:18]: - [ ] Great job on the functionality for adding and deleting but it shouldn't presist in the page so after deleting the tasks and creating a new task I can see them again in the page so please delete it and don't let the task presist in the page 😊🙏🏻🔥✨📌

File: src/index.b.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+  const html = `
+    <div class="header">
+    <div class="check-content">
+      <input type="checkbox" class="checkbox" />
+      <div class="task"><p>${task}</p></div>
+    </div>
+    <i class="fa-solid fa-ellipsis-vertical"></i>
+  </div>`;
+  taskContent.insertAdjacentHTML('beforeend', html);
```
Comment (Reem-lab) [inline src/index.b.js:18]: - [ ] I could see that you didn't add index to your tasks here so I recommend to add indexes (i.e. if you're adding a 5th task to the list, the index of that task should equal to 5). as required and please update the index after deleting a task (i.e. if you're deleting the first task index 1 from the list, the index of the next task(2) should set to 1)..🔥✨💯✅

**Review by Reem-lab — CHANGES_REQUESTED** [review]
# Hi @audukehinde, 👋🏻👋🏻 

Good job so far! 💯🔥♻
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- No Linters error ✅
- Followed GitHub flow ✅
- Great working on the design ✅
- Professional README ✅🔥
- Descriptive PR ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (audukehinde) [reply src/index.js:20]: Done. Thanks

File: src/index.b.js (hunk 2 above)
Comment (audukehinde) [reply src/index.b.js:18]: Done. Thanks

**Review by Lizdev-05 — APPROVED** [review]
Hi @audukehinde, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [ ] No Linters error ✅
- [ ] Followed GitHub flow ✅
- [ ] Great working on the design ✅
- [ ] Professional README ✅🔥
- [ ] Descriptive PR ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**